client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -335,6 +345,8 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 										</Button>
 										<Button
 											variant="primary"
+											isBusy={ isLoading }
+											disabled={ isLoading }
```

Comment:
Renaming `isLoading` would make this clearer IMO. E.g. 

```jsx
<Button
  label="submit"
  disabled={ isSubmitting }
/>
```

Reply:
Renamed to `isDisputeAcceptRequestPending` in b60c8dc.